src/worker.cc

```diff
@@ -1117,7 +1132,9 @@ void from_json(rapidjson::Document& doc, Options::Action action, Api& api) {
       throw valhalla_exception_t(144, *exp_action_str);
     }
     options.set_expansion_action(exp_action);
-  } else if (options.action() == Options_Action_expansion) {
+  }
+
+  if (options.action() == Options::expansion && !options.has_expansion_action()) {
```

Comment:
honestly, not sure that this is a correct way of handling this, but without this change it was not possible to use `/expansion` endpoint with `pbf`-encoded request